javalin-testtools/pom.xml

```diff
@@ -73,5 +73,9 @@
             <artifactId>jackson-module-kotlin</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
```

Comment:
The tests in `javalin-rendering` and `javalin-micrometer` were failing because Javalin could not find an ObjectMapper on startup. I am guessing that it before would get transitively included. Using `<scope>test</scope>` here did not work, I am not sure why.